packages/ra-ui-materialui/src/input/ArrayInput/SimpleFormIterator.stories.tsx

```diff
@@ -177,3 +177,18 @@ export const Sx = () => (
         </Edit>
     </AdminContext>
 );
+
+export const ClearButton = () => (
```

Comment:
There is no difference with the Basic story...

Reply:
Story changes to `<DisableClearButton />`, and display in case of `clearButton` prop is false.
However, this behavior pretty much similar to `<DisableRemove />` story.